src/components/molecules/PollBox/PollBox.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback, useMemo } from "react";
+import { useForm, useFieldArray, Controller } from "react-hook-form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaperPlane, faPlus } from "@fortawesome/free-solid-svg-icons";
+import { InputField } from "components/atoms/InputField";
+import { MAX_QUESTIONS_NUMBER } from "settings";
+import "./PollBox.scss";
+
+export type Question = {
+  name: string;
+};
+
+export type PollValues = {
+  topic: string;
+  questions: Question[];
+};
+
+export interface PollBoxProps {
+  onSubmit: (args: PollValues) => void;
+}
+
+const defaultQuestion: Question = { name: "" };
+const defaultValues = {
+  topic: "",
+  questions: [defaultQuestion, defaultQuestion],
+};
```

Comment:
We tend to use UPPER_SNAKE_CASE for constants like this:

```suggestion
const DEFAULT_QUESTION: Question = { name: "" };
const DEFAULT_VALUES = {
  topic: "",
  questions: [DEFAULT_QUESTION, DEFAULT_QUESTION],
};
```